### PR TITLE
Data streams

### PR DESCRIPTION
**Pull request by Karl-Cui**

### CODE REVIEW

File: neurostack/data_streams/data_stream.py [hunk 1]
```diff
@@ -1,38 +1,105 @@
...
+        # get channel names
+        ch_names = []
+        this_child = info.desc().child('channels').child('channel')
+        for _ in range(info.channel_count()):
+            ch_names.append(this_child.child_value('label'))
+            this_child = this_child.next_sibling('channel')
+        self.eeg_channel_names = ch_names
+
+        # record data to channels
+        self._thread = threading.Thread(target=self._record_lsl_data_indefinitely,
```
Comment (JunWeiWong) [inline neurostack/data_streams/data_stream.py]: Need to initialize threading before using

File: neurostack/data_streams/data_stream.py [hunk 2]
```diff
@@ -1,38 +1,105 @@
...
+                                        name='lsl')
+        self._thread.daemon = True
+        self._thread.start()
+
+    def _record_lsl_data_indefinitely(self):
```
Comment (JunWeiWong) [inline neurostack/data_streams/data_stream.py]: This method takes 'inlet' as arguement

File: neurostack/data_streams/data_stream.py (hunk 1 above)
Comment (Karl-Cui) [reply neurostack/data_streams/data_stream.py]: Fixed

File: neurostack/data_streams/data_stream.py (hunk 2 above)
Comment (Karl-Cui) [reply neurostack/data_streams/data_stream.py]: Fixed by adding variable to class

**Pull request merged**